### ISSUE TITLE
Themes not listed - only Default theme

### ISSUE THREAD

**Issue by winternet-studio** [benweier/Themr#18]
When I press Ctrl-F5 neither the built-in themes nor my own custom theme is listed - I only see the Default theme (which does nothing when I click on it). How come?

I'm using ST3 build 3126 in Windows 10.

**Comment by benweier** [comment]
Hi @winternet-studio have you installed any themes to be selected through the command palette?

I recommend trying the Boxy theme. Open the command palette, choose Package Control: Install Package, search for Boxy, and select it to install. Then ctrl+f5 to open your theme list.

**Comment by winternet-studio** [comment]
No, I just have the list of default themes that comes with ST3 and then my own custom theme. Are you saying it won't work with those?
